### PR TITLE
Fix: Cannot return null for non-nullable field Webhook.name.

### LINKED ISSUE #12989: Bug: Cannot return null for non-nullable field Webhook.name.
### What are you trying to achieve?

Open app in dashboard.

It seems that name is optional https://docs.saleor.io/docs/3.x/api-reference/webhooks/inputs/webhook-create-input#code-style-fontweight-normal-webhookcreateinputbnamebcodestring-

But querying such webhooks would return error

### Steps to reproduce the problem

```graphql
mutation{
  webhookCreate(input:{targetUrl:"https://examples.site/", syncEvents:[SHIPPING_LIST_METHODS_FOR_CHECKOUT], app:"QXBwOjY5"}){
    errors{
      message
      field
    }
    webhook{
      name
    }
  }
}
```

```graphql
query App($id: ID!) {
  app(id: $id) {
    ...App
    aboutApp
    author
    permissions {
      code
      name
      __typename
    }
    dataPrivacy
    dataPrivacyUrl
    __typename
  }
}

fragment App on App {
  id
  name
  created
  isActive
  type
  homepageUrl
  appUrl
  manifestUrl
  configurationUrl
  supportUrl
  version
  accessToken
  privateMetadata {
    key
    value
    __typename
  }
  metadata {
    key
    value
    __typename
  }
  tokens {
    authToken
    id
    name
    __typename
  }
  webhooks {
    ...Webhook
    __typename
  }
  __typename
}

fragment Webhook on Webhook {
  id
  name
  isActive
  app {
    id
    name
    __typename
  }
  __typename
}
```

```
{
  "id": "QXBwOjY5",
  "PERMISSION_MANAGE_USERS": true,
  "PERMISSION_MANAGE_STAFF": true,
  "PERMISSION_IMPERSONATE_USER": true,
  "PERMISSION_MANAGE_APPS": true,
  "PERMISSION_MANAGE_OBSERVABILITY": true,
  "PERMISSION_MANAGE_CHECKOUTS": true,
  "PERMISSION_HANDLE_CHECKOUTS": true,
  "PERMISSION_HANDLE_TAXES": true,
  "PERMISSION_MANAGE_TAXES": true,
  "PERMISSION_MANAGE_CHANNELS": true,
  "PERMISSION_MANAGE_DISCOUNTS": true,
  "PERMISSION_MANAGE_GIFT_CARD": true,
  "PERMISSION_MANAGE_MENUS": true,
  "PERMISSION_MANAGE_ORDERS": true,
  "PERMISSION_MANAGE_PAGES": true,
  "PERMISSION_MANAGE_PAGE_TYPES_AND_ATTRIBUTES": true,
  "PERMISSION_HANDLE_PAYMENTS": true,
  "PERMISSION_MANAGE_PLUGINS": true,
  "PERMISSION_MANAGE_PRODUCTS": true,
  "PERMISSION_MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES": true,
  "PERMISSION_MANAGE_SHIPPING": true,
  "PERMISSION_MANAGE_SETTINGS": true,
  "PERMISSION_MANAGE_TRANSLATIONS": true
}
```

```
{
  "errors": [
    {
      "message": "Cannot return null for non-nullable field Webhook.name.",
      "locations": [
        {
          "line": 55,
          "column": 3
        }
      ],
      "path": [
        "app",
        "webhooks",
        1,
        "name"
      ],
      "extensions": {
        "exception": {
          "code": "GraphQLError"
        }
      }
    }
  ],
  "data": {
    "app": {
      "id": "QXBwOjY5",
      "name": "Manage products",
      "created": "2023-03-27T15:39:57.879720+00:00",
      "isActive": true,
      "type": "LOCAL",
      "homepageUrl": null,
      "appUrl": null,
      "manifestUrl": null,
      "configurationUrl": null,
      "supportUrl": null,
      "version": null,
      "accessToken": null,
      "privateMetadata": [],
      "metadata": [],
      "tokens": [
        {
          "authToken": "Myd1",
          "id": "QXBwVG9rZW46Njk=",
          "name": "Default",
          "__typename": "AppToken"
        },
        {
          "authToken": "Dpqk",
          "id": "QXBwVG9rZW46NzA=",
          "name": "manage",
          "__typename": "AppToken"
        }
      ],
      "webhooks": null,
      "__typename": "App",
      "aboutApp": null,
      "author": null,
      "permissions": [
        {
          "code": "HANDLE_CHECKOUTS",
          "name": "Handle checkouts",
          "__typename": "Permission"
        },
        {
          "code": "HANDLE_PAYMENTS",
          "name": "Handle payments",
          "__typename": "Permission"
        },
        {
          "code": "HANDLE_TAXES",
          "name": "Handle taxes",
          "__typename": "Permission"
        },
        {
          "code": "IMPERSONATE_USER",
          "name": "Impersonate user.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_APPS",
          "name": "Manage apps",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_CHANNELS",
          "name": "Manage channels.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_CHECKOUTS",
          "name": "Manage checkouts",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_DISCOUNTS",
          "name": "Manage sales and vouchers.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_GIFT_CARD",
          "name": "Manage gift cards.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_MENUS",
          "name": "Manage navigation.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_OBSERVABILITY",
          "name": "Manage observability",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_ORDERS",
          "name": "Manage orders.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_PAGES",
          "name": "Manage pages.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_PAGE_TYPES_AND_ATTRIBUTES",
          "name": "Manage page types and attributes.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_PLUGINS",
          "name": "Manage plugins",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_PRODUCTS",
          "name": "Manage products.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES",
          "name": "Manage product types and attributes.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_SETTINGS",
          "name": "Manage settings.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_SHIPPING",
          "name": "Manage shipping.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_STAFF",
          "name": "Manage staff.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_TAXES",
          "name": "Manage taxes",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_TRANSLATIONS",
          "name": "Manage translations.",
          "__typename": "Permission"
        },
        {
          "code": "MANAGE_USERS",
          "name": "Manage customers.",
          "__typename": "Permission"
        }
      ],
      "dataPrivacy": null,
      "dataPrivacyUrl": null
    }
  },
  "extensions": {
    "cost": {
      "requestedQueryCost": 3,
      "maximumAvailable": 50000
    }
  }
}
```

### What did you expect to happen?

Return null

### Logs

_No response_

### Environment

Saleor version: 3.13

### PR DESCRIPTION
**Pull request by cyborg7898**
I want to merge this change because 
It closes #12989 

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

**Review by maarcingebala — APPROVED** [review]

**Comment by maarcingebala** [comment]
@cyborg7898 Please generate a new GraphQL schema using `npm run build-schema` and add it to the PR.

**Review by fowczarek — COMMENTED** [review]
Can you add test for that? e.b `test_query_webhook_without_name`?

**Comment by cyborg7898** [comment]
@fowczarek I've added the test case but unfortunately I'm unable to test the test cases. Getting error like ` django.db.utils.ProgrammingError: function gen_random_uuid() does not exist` even  `gen_random_uuid()` is returning uuid in postgres command line. Also tried in psql `CREATE EXTENSION pgcrypto;` but showing already exist. 
Please let me know if test case is breaking. Thanks!

**Comment by cyborg7898** [comment]
@fowczarek Just tested the above test case it's working fine.

**Review by fowczarek — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ### Other changes
 - Expand metric units to support more types of products. - #13043 by @FremahA
-
+- Fix Error Cannot return null for non-nullable field Webhook.name. - #12989 by @cyborg7898
```
Comment (maarcingebala) [inline CHANGELOG.md:15]:
```suggestion
- Fix Error Cannot return null for non-nullable field Webhook.name. - #12989 by @cyborg7898

```

**Comment by maarcingebala** [comment]
Thanks and congrats on first PR to Saleor @cyborg7898 🎉

**Pull request merged**